server/src/main/java/org/opensearch/indices/recovery/PeerRecoveryStatsTracker.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices.recovery;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Tracker responsible for computing PeerRecoveryStats.
+ *
+ * @opensearch.internal
+ */
+public class PeerRecoveryStatsTracker {
+    private final AtomicLong totalStartedRecoveries;
+    private final AtomicLong totalFailedRecoveries;
+    private final AtomicLong totalCompletedRecoveries;
+    private final AtomicLong totalRetriedRecoveries;
+    private final AtomicLong totalCancelledRecoveries;
+
+    public PeerRecoveryStatsTracker() {
+        totalStartedRecoveries = new AtomicLong();
+        totalFailedRecoveries = new AtomicLong();
+        totalCompletedRecoveries = new AtomicLong();
+        totalRetriedRecoveries = new AtomicLong();
+        totalCancelledRecoveries = new AtomicLong();
+    }
+
+    public void incrementTotalStartedRecoveries(long increment) {
+        totalStartedRecoveries.addAndGet(increment);
+    }
```

Comment:
If we want to support adding >1 value, the convention in the repo is to name the methods `add*()`. `increment*()` methods in the repo do not take an argument and enforce addition of 1.